Hypnotic-Herons/Sprint-2
# added department views form

# Steps to Test
Outline the steps to test

```
git fetch --all
git checkout js_createdepartment
code .
```

1. navigate to the root
1. python manage.py makemigrations HRIT
1. python manage.py migrate
1. python manage.py createsuperuser (this is just for your local use)
1. python manage.py runserver
1. login on the admin view http://127.0.0.1:8000/admin
1. add a few departments
1. look at the browser view for department to see the list http://127.0.0.1:8000/HRIT/departments/
1. click on "add a department" and you should see a form to add department
1. click submit and you should see the department you just created in the department list
